src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (num = null) => {
+    if (num === null) {
```

Comment:
you can use just the check for `undefined` without setting the default value. but your solution is good as well